apps/ledger-live-mobile/src/newArch/features/Web3Hub/screens/Web3HubMain/components/ManifestsCategoryList/index.tsx

```diff
@@ -34,7 +34,7 @@ const ManifestsCategoryList = ({ title, categoryId, navigation }: Props) => {
       <HorizontalList
         title={title}
         data={data}
-        extraData={{ onPress: disclaimer.onPressItem }}
+        extraData={disclaimer.onPressItem}
```

Comment:
This prop should probably be called `onPressItem` as we don't really want to say that we expose `extraData` especially because nothing else than a function would work here